src/index.js

```diff
@@ -0,0 +1,64 @@
+import _ from 'lodash';
+import './style.css';
+
+// Selectors
+const todoInput = document.querySelector('.todo-input');
+const todoButton = document.querySelector('.todo-button');
+const todoList = document.querySelector('.todo-list');
+
+// Functions
+
+const addTodo = (event) => {
+  // Prevent form from submitting
+  event.preventDefault();
+  // Todo div
+  const todoDiv = document.createElement('div');
+  todoDiv.classList.add('todo');
+
+  // Checked mark button
+  const completedButton = document.createElement('button');
+  completedButton.innerHTML = '<i class="fas fa-check"></i>';
+  completedButton.classList.add('complete-btn');
+
+  todoDiv.appendChild(completedButton);
+
+  // Create Li
+  const newTodo = document.createElement('li');
+  newTodo.innerText = 'Hard coded To do Items';
+  newTodo.classList.add('todo-item');
```

Comment:
- [ ] You should use a populated  class to add the tasks and it should take the input as argument

Reply:
## @dasileker thank you for the feedback.